pkgs/tools/networking/connman/connman.nix

```diff
@@ -67,16 +67,17 @@ stdenv.mkDerivation rec {
     libmnl
     gnutls
     readline
-  ] ++ optionals (enableOpenconnect) [ openconnect ];
+  ] ++ optionals (enableOpenconnect) [ openconnect ]
+    ++ optionals (firewallType == "iptables") [ iptables ]
+    ++ optionals (firewallType == "nftables") [ libnftnl ]
+  ;
 
   nativeBuildInputs = [
     pkg-config
     file
   ]
     ++ optionals (enablePolkit) [ polkit ]
     ++ optionals (enablePptp) [ pptp ppp ]
```

Comment:
Now that I'm thinking about it, it's a bit surprising these are not in `buildInputs`.. Could you share the `nix build` command you tried to run that without the patch?

Reply:
Now I noticed you fixed the cross compilation of `connmanMinimal` which doesn't enable these features by default. Perhaps that can be handled in the future. Could you please @alyssais make the commit message a bit more informative?

Reply:
More informative how?  It's obvious from the diff I moved stuff from `nativeBuildInputs` to `buildInputs`, so the only thing I could explain is the difference between `buildInputs` and `nativeBuildInputs`, but it doesn't make sense to do that for every single commit I do that fixes cross-compilation — that's what the manual's for.